k8s/aat/common/em/em-stitching.yaml

```diff
@@ -26,6 +26,8 @@ spec:
       image: hmctspublic.azurecr.io/em/stitching:prod-464c8e27
       ingressHost: em-stitching-aat.service.core-compute-aat.internal
       environment:
+        SPRING_DATASOURCE_URL: jdbc:postgresql://em-stitching-postgres-db-aat.postgres.database.azure.com:5432/stitching
```

Comment:
this should really be set in the application repo:
https://github.com/hmcts/rpa-em-stitching-api/blob/master/charts/em-stitching/values.yaml#L25

preview configuration can be overridden in values.preview.template.yaml

Reply:
I set it in values.aat.template.yaml instead or is this the incorrect place?

Reply:
done

Reply:
it should be in values.yaml